alot/helper.py

```diff
@@ -10,6 +10,7 @@
 from datetime import datetime
 from collections import deque
 from io import BytesIO
+from cStringIO import StringIO
```

Comment:
Sorry I didn't get to this sooner, I was taking a vacation day yesterday.

Could we use StringIO.StringIO instead of the cStringIO variant which can't represent unicode that isn't also ascii (utf-8 is a superset of base ascii)?